consensus/module.go

```diff
@@ -34,6 +35,8 @@ type ConsensusModule struct {
 	privateKey cryptoPocket.Ed25519PrivateKey
 	consCfg    modules.ConsensusConfig
 
+	m sync.RWMutex
```

Comment:
Can you add a comment about why its needed and where/how it should be used?

Reply:
Maybe addressed in https://github.com/pokt-network/pocket/pull/198/commits/5c87b153042b43b85152faeccde815e02f9748d5

Reply:
I have some other related questions, but this definitely addresses anything a reader should need to know. No more & no less.